pkg/buildkite/resources.go

```diff
@@ -0,0 +1,230 @@
+package buildkite
+
+import (
+	"context"
+
+	"github.com/mark3labs/mcp-go/mcp"
+)
+
+func HandleDebugLogsGuideResource(ctx context.Context, request mcp.ReadResourceRequest) ([]mcp.ResourceContents, error) {
+	content := `# Debugging Buildkite Build Failures with Job Logs
```

Comment:
This is a bit of an optional nitpick. I wonder if we should `//go:embed` this from a markdown file?

https://pkg.go.dev/embed

Mostly to make iterating on it happen a bit more easily / be able to look at it rendered in human friendly formatting.

Reply:
@catkins done.